fontc_crater/README.md

```diff
@@ -7,13 +7,16 @@ fonts.
 
 
 ```sh
-$ cargo run --release -p=fontc_crater -- compile FONT_CACHE --fonts-repo GOOGLE/FONTS  -o results.json
-```
+# By default font repositories and results will be written to ./font_repos
```

Comment:
The repo cache is expected to be transient, I wouldn't want to write anything important there..

Reply:
I think if I address the don't default results there this is addressed?